### PR TITLE
Bench darwinia-account-migration

### LINKED ISSUE #971: `account-migration` benchmark

### PR DESCRIPTION
**Pull request by AurevoirXavier**
Closes #971.

### CODE REVIEW

File: pallet/account-migration/src/benchmarking.rs [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+				balance: kton,
+				is_frozen: Default::default(),
+				reason: ExistenceReason::Sufficient,
+				extra: Default::default(),
+			},
+		);
+		<Vestings<T>>::insert(
+			&from,
+			vec![
+				VestingInfo::new(Default::default(), Default::default(), Default::default());
```
Comment (boundless-forest) [inline pallet/account-migration/src/benchmarking.rs:67]: Default value cause `set_lock` early return.


```rs
fn set_lock() {
    if amount.is_zero() || reasons.is_empty() {
			return
	}
}
```

File: pallet/account-migration/src/benchmarking.rs [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+					(
+						Default::default(),
+						Default::default()
+					);
+					<T as darwinia_staking::Config>::MaxUnstakings::get()
+						as usize
+				]),
+				unstaking_kton: BoundedVec::truncate_from(vec![
+					(
+						Default::default(),
```
Comment (boundless-forest) [inline pallet/account-migration/src/benchmarking.rs:120]: And this.

File: pallet/account-migration/src/benchmarking.rs [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+				staked_ring: Default::default(),
+				staked_kton: kton,
+				staked_deposits: BoundedVec::truncate_from(vec![
+					Default::default();
+					<T as darwinia_deposit::Config>::MaxDeposits::get()
+						as usize
+				]),
+				unstaking_ring: BoundedVec::truncate_from(vec![
+					(
+						Default::default(),
```
Comment (boundless-forest) [inline pallet/account-migration/src/benchmarking.rs:112]: This

File: pallet/account-migration/src/benchmarking.rs (hunk 2 above)
Comment (AurevoirXavier) [reply pallet/account-migration/src/benchmarking.rs:120]: What do you mean "and this"?

Comment (boundless-forest) [reply pallet/account-migration/src/benchmarking.rs:120]: See https://github.com/darwinia-network/darwinia/pull/1029#discussion_r1139555703. The current version of benchmark does not cover the balance transfer and asset transafer cases.

Comment (AurevoirXavier) [reply pallet/account-migration/src/benchmarking.rs:120]: See `staked_kton += 1;`. Assets transfer is calculated.

**Pull request merged**